client/src/js/directives/findDebtorGroup.js

```diff
@@ -0,0 +1,117 @@
+/**
+* <bh-find-debtorgroup></bh-find-debtorgroup>
+*
+* The find debtor group component is responsible for displaying a component
+* for searching a debtor group or a convention
+*
+* This component takes these attributes :
+*   - is-onvention : if we want only debtor groups which are conventions
+*   - on-search-complete : the callback function which get the returned debtror group
+*
+* @example
+* // component find debtor group
+* <bh-find-debtorgroup>
+*   is-convention="true"
+*   on-search-complete="MyCtrl.handleResult"
+* </bh-find-debtorgroup>
+*/
+
+/** Find Debtor Group Controller */
+function findDebtorGroupController($http) {
+  var ctrl         = this,
+      callback     = this.callback,
+      locked       = this.locked,
+      isConvention = this.isConvention;
+
+  /** Load debtor group */
+  loadgDebtorGroup();
```

Comment:
Why is there an extra g in this function name?  Is there something wrong with `loadDebtorGroup()`?